common/windows-app-bootstrap.cpp

```diff
@@ -5,6 +5,10 @@
 // We need WinMain on Windows to offer proper Windows application and not console application
 // Should not be used in CMake on Linux
 
+#ifndef NOMINMAX
```

Comment:
Don't need the `#ifndef` since we never included anything...

Reply:
done